templates/block/asu-footer--footer-block.html.twig

```diff
@@ -155,7 +155,27 @@
               class="collapse card-body"
               aria-labelledby="footlink-header-{{ key }}-{{ loop.index }}"
             >
-              {{ c.menu_items }}
+              {# WS2-1268: Adding block title to data-ga-footer-component attribute.
+               Building menu items here to access block title, rather than in AsuFooterBlock.php
+               TODO: Rewrite this module as UDS react components similar to the ASU Header
```

Comment:
@chill5-0 More to the point of what I was getting at by suggesting these updates need to be made in UDS  is that we need to make sure any markup changes that are made here also end up in the UDS `component-footer` (sub-components here https://github.com/ASU/asu-unity-stack/tree/dev/packages/component-footer/src/components) and the `bootstrap4-theme` component (https://github.com/ASU/asu-unity-stack/blob/dev/packages/bootstrap4-theme/stories/organisms/global-footer/global-footer.templates.js) because those are the upstream sources of truth. 

These changes aren't unique to the WS2 CMS and should be reflected at the canonical source (UDS), unless I'm missing something.